Packs/Base/Scripts/DBotSuggestClassifierMapping/DBotSuggestClassifierMapping.yml

```diff
@@ -0,0 +1,44 @@
+args:
```

Comment:
Perhaps add more initial values/list options to parameters?

Reply:
Not really needed, it's script that the server run internally, no one should run it manually, this is why is depracated.